tests/acceptance/contracts/ThresholdSigningCohortMultisig.sol

```diff
@@ -97,67 +99,72 @@ contract ThresholdSigningCohortMultisig is
     /**
      * @notice Get unsigned hash for transaction parameters
      * @dev Follows ERC191 signature scheme: https://github.com/ethereum/EIPs/issues/191
-     * @param _sender Trustee who will execute the transaction
-     * @param _destination Destination address
-     * @param _value Amount of ETH to transfer
-     * @param _data Call data
-     * @param _nonce Nonce
+     * @param sender Trustee who will execute the transaction
```

Comment:
Not sure if this is on purpose, but this `ThresholdSigningCohortMultisig.sol` implementation is different to the nucypher-contracts PR #366 one: https://github.com/nucypher/nucypher-contracts/pull/366/files#diff-c7dcb42b7fecbaa0deeab5788b707c2fdc02f763fa3bea16e5156e133c746658

Reply:
It is on purpose.

There is a bug with ape that causes issues when extending a contract from a dependency - https://github.com/ApeWorX/ape/issues/2610. So to work around it for now, it is basically the same base as what is in `nucypher-contracts` but has some additional functions for theories/musings about what we could do in the future.